js_files/index.js

```diff
@@ -4,10 +4,23 @@ document.addEventListener("keyup", (e) => {
     Calculator();
   }
 });
-function toggle(ele) {
-  var cont = document.getElementById("cont");
-  cont.style.display = cont.style.display == "none" ? "block" : "none";
-}
+const calender=document.getElementById("cont");
+let calender1=window.getComputedStyle(calender).getPropertyValue("display");
+const log=document.getElementById("btn1");
+
+console.log(calender1);
+console.log(log);
+log.addEventListener("click",(e)=>{
+  console.log(calender1);
```

Comment:
delete all console log code from the file. It's not a good practice to put console code